src/applications/ivc-champva/10-10D/config/form.js

```diff
@@ -132,6 +132,21 @@ import { fileWithMetadataSchema } from '../../shared/components/fileUploads/atta
 // import mockData from '../tests/e2e/fixtures/data/test-data.json';
 import FileFieldWrapped from '../components/FileUploadWrapper';
 
+// Control whether we show the file overview page by calling `hasReq` to
+// determine if any files have not been uploaded
+function showFileOverviewPage(formData) {
+  try {
+    return (
+      hasReq(formData.applicants, true, true) ||
+      hasReq(formData.applicants, false, true) ||
+      hasReq(formData, true, true) ||
+      hasReq(formData, false, true)
+    );
+  } catch {
+    return false;
+  }
+}
+
```

Comment:
Pulled this up to top of file and wrapped in function so it can more concisely be used on the two pages that require this check.

This is the same depends functionality we previously used, but now the optional third parameter passed to each `hasReq` usage is set to true, causing us to ignore uploaded functions. This was missing previously, causing our depends function to return true when a user had uploaded all their files.

It now appropriately returns false. This was a usage error and not a bug proper.